code-challenges.js

```diff
@@ -7,43 +7,125 @@
-const myNumbers2 = [8, -7, 42, 9, 13]
 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 eilrahC"
+// create a function that will reverse the characters inside a string.
+// const currentCohort = "Charlie 2022"
+
 
-const currentCohort = "Charlie 2022"
 
+// var currentCohort = ["C", "h", "a", "r", "l","i", "e"," ", "2", "0", "2", "2"]  // To poppulat the array
+// var curCohortReversed = currentCohort.reverse()                                 // using reverse function to reverse order of characters in the array
+// console.log(curCohortReversed.join(""))                                         // using join function for the output to join characters together.
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
+//  create a function using modolo to evaluate whether a variable is odd or even.
+// const myArray = [13, 34, 5, 10, 27, 42]
+// 
+
+ //create a function that declares an array as argument
+//  var numArray = [12, 34, 5, 10, 27, 42]
+//  const findOddEven = (arr) => {
+//  //create an array to store values of add or even
+// let oddOrEven = []
+// //create loop going through the array to pass the function
+//   for(let currentVal = 0; currentVal<arr.length; currentVal++) {
+//       // use conditional to evaluate if even
+//       if(arr(currentVal) % 2 ===0) {
+//           //push the value "even" to the oddOrEven array
+//           oddOrEven.push[even]
+
+//          }
+//             // other wise, push odd to the oddOrEven array
+//              else  { oddOrEven.push[odd]    
+//             }
+//             //return oddOrEven array
+//             return oddOrEven
+//              }
+//         // to call the function               
+//            console.log(findOddEven(numArray))    
+        
+  
+
+
+
+
+
+
 
-const myArray = [13, 34, 5, 10, 27, 42]
 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
 // Set one:
-const number1 = 58
-const number2 = 100
+// const number1 = 58
+// const number2 = 100
 
 // Set two:
-const number3 = 27
-const number4 = 24
+// const number3 = 27
+// const number4 = 24
+//create function smallDiff  with arguments num1 and num2
+// const smallDiff = (num1, num2) => {
+//  // compare the two numbers
+//     if num1 > num2
+//     // if num1 is bigger, deduct num2 from num 1 and return the difference
+//       return `$(num1) - (num2)` }
+//     // otherwise deduct num1 from num2 and return the difference
+//       else  { return `$(num2) - (num1)` }
+// //invoke the function
+//       console.log(smallDiff(58,100));
+//       console.log(smallDiff(27, 24))
```

Comment:
Good stuff! Your pseudo code tells you to make a function and I think if you spend some time going back over this that would be a good refactor!